src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Collections.Generic;
 using System.Net;
+using Stratis.Bitcoin.Connection;
```

Comment:
why this?

Reply:
Removed. It was left over after some optimization was reverted.